icons/circle-dot-draft.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="10" style="stroke-dasharray:3.9252,3.9252" />
```

Comment:
We don't support inline styles, so this needs to be flattened:
```suggestion
  <path d="M4.26 5.67c.42-.52.9-1 1.41-1.41" />
  <path d="M2.05 13a9.93 9.93 0 0 1 0-2" "/>
  <path d="M5.67 19.74a10 10 0 0 1-1.41-1.41" />
  <path d="M13 21.95c-.66.07-1.33.07-2 0" />
  <path d="M19.74 18.33c-.42.52-.9 1-1.41 1.41" />
  <path d="M21.95 11c.07.66.07 1.33 0 2" />
  <path d="M18.33 4.26c.52.42 1 .9 1.41 1.41" />
  <path d="M11 2.05a9.93 9.93 0 0 1 2 0" />
```

The name should probably also be something like `circle-dashed-dot`, and it would be neat to also add the regular `circle-dashed`.

Reply:
Can it not be flattened so that the gap between each dash is 2px like before?

Reply:
@karsa-mistmere Can it not be flattened so that the gap between each dash is 2px like before?

Reply:
> @karsa-mistmere Can it not be flattened so that the gap between each dash is 2px like before?

Never mind, I figured out a way…